src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -892,6 +757,17 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
         return handled;
     }
 
+    // Method Description:
+    // - handle a tap event by taking focus
+    // Arguments:
+    // - sender: the XAML element responding to the tap event
+    // - args: event data
+    void TermControl::_TappedHandler(const IInspectable& /*sender*/, const TappedRoutedEventArgs& e)
+    {
+        Focus(FocusState::Pointer);
+        e.Handled(true);
+    }

```

Comment:
I wonder if introducing this event should make us change how we handle a "tap" for selection. @leonMSFT thoughts?

[Doc](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.uielement.tapped)

Reply:
This event was _not_ changed or added; it was lifted out of a lambda into its own function.